services/gitdiff/gitdiff.go

```diff
@@ -180,55 +181,61 @@ var (
 	removedCodePrefix = []byte(`<span class="removed-code">`)
 	codeTagSuffix     = []byte(`</span>`)
 )
+var re = regexp.MustCompile(`<span class="[a-z]*$`)
```

Comment:
It should be named more specific as it is global variable

Reply:
right : ) fixed